app/scripts/modules/core/src/managed/artifactDetail/verifications/VerificationCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { IVerification } from '../../../domain';
```

Comment:
All the top level modules have aliases in webpack and typescript that should be used.
There are linter rules that I think should have flagged this 🤔 


```suggestion
import { IVerification } from 'domain/IManagedEntity';
```

Reply:
welp, looks like I spoke untrue words.  in fact, `domain` isn't aliased but `core` is and should be accessed at `core/domain`

```
import { IVerification } from 'core/domain/IManagedEntity';

```